File: SEproject/Data/Course.cs
using System;
using System.Collections.Generic;

namespace SEproject.Data
{
    public partial class Course
    {
        public string CourseId { get; set; } = null!;
        public string CourseName { get; set; } = null!;
        public string? InstructorId { get; set; }
        public string? InstructorFirst { get; set; }
        public string? InstructorLast { get; set; }
        public string? InstructorEmail { get; set; }
    }
}

File: SEproject/ClientApp/src/app/data/models/course.ts
export class Course {
    courseID: string;
    courseName: string;
    instructorID: string;
    instructorEmail: string;
    instructorFirst: string;
    instructorLast: string;

    constructor(id: string, courseName: string, instructorID: string, instructorFirst: string, instructorLast: string, instructorEmail: string) {
        this.courseID = id;
        this.courseName = courseName;
        this.instructorID = instructorID;
        this.instructorFirst = instructorFirst;
        this.instructorLast = instructorLast;
        this.instructorEmail = instructorEmail;
    }
}
